resources/authgear/templates/zh-HK/translation.json

```diff
@@ -0,0 +1,229 @@
+  "settings-page-recovery-code-title": "Recovery Code",
+  "settings-page-secondary-oob-otp-title": "以SMS / Email獲取驗證碼",
+  "settings-page-secondary-password-description": "上次更新時間: {time, datetime, long}",
+  "settings-page-secondary-password-title": "輔助密碼",
+  "settings-page-secondary-totp-title": "以驗證器應用程式 / 裝置獲取驗證碼",
+  "settings-page-security-section-description": "用於確保登入的是你本人",
+  "settings-page-security-section-title": "安全",
+  "settings-page-session-section-description": "管理你已登入的裝置",
+  "settings-page-session-section-title": "已登入的裝置",
+  "settings-page-trusted-devices-title": "Trusted Devices",
+  "settings-sessions-description": "你目前在以下裝置登入",
+  "settings-sessions-item-description": "最後活動: {time, datetime, long}",
+  "settings-sessions-revoke-all-label": "登出所有裝置",
+  "settings-sessions-revoke-label": "登出",
+  "settings-sessions-title": "已登入的裝置",
+  "settings-totp-item-description": "Added at {time, datetime, long}",
+  "settings-totp-title": "Authenticator Apps / Devices",
+  "setup-button-label": "Setup",
+  "setup-oob-otp-instead": "Setup SMS / email 2-step verification instead",
+  "setup-oob-otp-title--email": "Setup email 2-step verification",
+  "setup-oob-otp-title--sms": "Setup SMS 2-step verification",
+  "setup-oob-otp-use-email-description": "Setup email 2-step verification instead",
+  "setup-oob-otp-use-phone-description": "Setup SMS 2-step verification instead",
+  "setup-password-button-label": "設置密碼",
+  "setup-password-instead": "改為設置密碼",
+  "setup-totp-code-description": "Enter the 6-digit code that you see in the app / device",
+  "setup-totp-code-placeholder": "code",
+  "setup-totp-fallback-description": "Can''t scan the barcode?\nChoose <b>Manual entry</b>.\nEnter your email address and this key:\n<b>{secret}</b>\nMake sure that Time based is turned on",
+  "setup-totp-get-google-authenticator-description": "If you don''t have any authenticator device / app, you can get the Google Authenticator App from <a class=\"link\" target=\"_blank\" href=\"https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2\">Google Play Store</a> or <a class=\"link\" target=\"_blank\" href=\"https://apps.apple.com/app/google-authenticator/id388497605\">Apple App Store</a>",
+  "setup-totp-instead": "Setup authenticator app / device instead",
+  "setup-totp-title": "Setup authenticator",
+  "setup-totp-use-google-authenticator-description": "In the App select <b>Set up account</b>.\nChoose <b>Scan barcode</b>",
+  "show-password-label": "顯示密碼",
+  "show-recovery-code-button-label": "顯示代碼",
+  "signup-label": "註冊",
+  "sms.default.sender": "Authgear",
+  "sso-login-id-separator-both-present": "或",
+  "sso-login-id-separator-oauth-present": "選擇以下一個選項以繼續",
+  "terms-of-service-link": "https://kurious.live/zh/terms-and-conditions",
+  "toc-pp-footer": "註冊即表示我已同意 <a target=\"_blank\" href=\"{termsOfService}\">條款和細則</a> 及 <a target=\"_blank\" href=\"{privacyPolicy}\">隱私政策</a>.",
```

Comment:
Although we support theme primary color for anchor without class. We may want to sync the class in 2 different locale, maybe remove the class in EN version?

Reply:
I will add the class to the zh version.